lms/envs/common.py

```diff
@@ -1217,6 +1217,9 @@ def _make_locale_paths(settings):
 MIDDLEWARE_CLASSES = [
     'crum.CurrentRequestUserMiddleware',
 
+    # Clears request cache. This is a safer request cache.
+    'edx_django_utils.cache.middleware.RequestCacheMiddleware',
+
     'openedx.core.djangoapps.request_cache.middleware.RequestCache',
```

Comment:
It is until and unless we do a clean-up to switch all of edx-platform over to use the newer RequestCache which would take some work, since it has a different contract (i.e. the CacheResponse).  I think we should do that, but we have a lot we should do. :(